### PR TITLE
Lendemor/templates radix sidebar

### PR DESCRIPTION
**Pull request by Lendemor**

### CODE REVIEW

**Review by masenf — APPROVED** [review]
Menu button seems a little too big but overall good change

image

**Review by masenf — CHANGES_REQUESTED** [review]
actually can you fix the broken links in README.md as well?

**Review by masenf — DISMISSED** [review]

File: reflex/.templates/apps/sidebar/code/styles.py [hunk 1]
```diff
@@ -5,10 +5,10 @@
 border_radius = "0.375rem"
 box_shadow = "0px 0px 0px 1px rgba(84, 82, 95, 0.14)"
 border = "1px solid #F4F3F6"
-text_color = "black"
-accent_text_color = "#1A1060"
-accent_color = "#F5EFFE"
-hover_accent_color = {"_hover": {"color": accent_color}}
+text_color = "var(--text-1)"
+accent_text_color = "var(--accent-10)"
```
Comment (picklelo) [inline reflex/.templates/apps/sidebar/code/styles.py:9]: I think you can do `rx.color("accent", 10)`

File: reflex/.templates/apps/sidebar/code/styles.py [hunk 2]
```diff
@@ -36,17 +36,15 @@
...
         **overlapping_button_style,
     },
-    rx.chakra.MenuItem: hover_accent_bg,
+    rx.menu.item: hover_accent_bg,
 }
 
 markdown_style = {
-    "code": lambda text: rx.chakra.code(text, color="#1F1944", bg="#EAE4FD"),
-    "a": lambda text, **props: rx.chakra.link(
+    "code": lambda text: rx.code(text, color="#1F1944", bg="#EAE4FD"),
```
Comment (picklelo) [inline reflex/.templates/apps/sidebar/code/styles.py:46]: These should also use the accent color

**Review by picklelo — CHANGES_REQUESTED** [review]

File: reflex/.templates/apps/sidebar/code/styles.py (hunk 1 above)
Comment (masenf) [reply reflex/.templates/apps/sidebar/code/styles.py:9]: you can _now_, that was literally added days ago 😁 #2713

**Review by picklelo — DISMISSED** [review]

File: reflex/.templates/apps/sidebar/code/pages/settings.py [hunk 3]
```diff
@@ -12,11 +12,50 @@ def settings() -> rx.Component:
...
         The UI for the settings page.
     """
-    return rx.chakra.vstack(
-        rx.chakra.heading("Settings", font_size="3em"),
-        rx.chakra.text("Welcome to Reflex!"),
-        rx.chakra.text(
+    return rx.vstack(
+        rx.heading("Settings", size="8"),
+        rx.hstack(
+            rx.text("Light mode: "),
```
Comment (picklelo) [inline reflex/.templates/apps/sidebar/code/pages/settings.py:18]: nit: this should be called "Dark mode" since switching it on makes it dark

**Review by picklelo — APPROVED** [review]

**Pull request merged**